build.gradle

```diff
@@ -1,18 +1,16 @@
 buildscript {
     ext.versions = [
         'incap' : '0.2',
-        'kotlin': '1.3.0'
+        'kotlin': '1.3.50'
     ]
 
     ext.deps = [
         android: [
             'runtime': 'com.google.android:android:4.1.1.4',
-            'gradlePlugin': "com.android.tools.build:gradle:3.2.1",
+            'gradlePlugin': "com.android.tools.build:gradle:3.5.1",
```

Comment:
we'll need to add `jcenter()` so we can pull down `trove4j` iirc

Reply:
I'm very confused about why my local builds work. I've deleted my dependency cache and everything and it still works fine...